volatility3/framework/plugins/windows/skeleton_key_check.py

```diff
@@ -344,12 +384,17 @@ def _analyze_cdlocatecsystem(self, function_bytes: bytes,
                 # we find the count before, so we can terminate the static analysis here
                 break
 
-        return array_start, count
+        if array_start and count:
+            array = self._construct_ecrypt_array(array_start, count, cryptdll_types)
+        else:
+            array = None
+
+        return array
 
     def _find_csystems_with_export(self, proc_layer_name: str, 
                                          cryptdll_types: interfaces.context.ModuleInterface, 
                                          cryptdll_base: int, 
-                                         _) -> Tuple[int, None, None]:
+                                         _) -> interfaces.context.ModuleInterface:
```

Comment:
If this can return `None`, it should be `Optional[interfaces.context.ModuleInterface]` although, I'm not sure it does return a `ModuleInterface`?